### PR TITLE
Version 42

### PR DESCRIPTION
**Pull request by vinniefalco**
* Fix javadoc typo
* Add formal review notes
* Make buffers_view a public interface

Documentation preview:
http://vinniefalco.github.io/stage/beast/v42

### CODE REVIEW

File: doc/review.qbk [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+[[
+    "Beast does not offer an HTTP server?"
+][
+    The scope of the library is to provide a message container,
+    and serialization and deserialization algorithms, with an
+    interface permitting users to achieve all the behaviors
+    necessary to implement high performance, conforming servers.
+    However, management of listening sockets, io_service threads,
+    individual connections, and HTTP semantics such as pipelining,
+    redirection, Expect: 100-continue, cookies, request routing,
```
Comment (nbougalis) [inline doc/review.qbk]: You probably want "and" after "cookies" and also ditch the comma after "request routing"

File: doc/review.qbk [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+    are not handled by Beast.
+    It is the author's position that keeping the library low-level
+    will establish it as a building block which the greater C++
+    community at large can leverage to produce higher level
+    libraries; competing with each other to figure out the best
```
Comment (nbougalis) [inline doc/review.qbk]: The semicolon here is a bit weird. A comma seems better suited.

File: doc/review.qbk [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+    of WebSocket processing have been optimized, and the HTTP parser design
+    is lifted from another extremely popular project which has performance
+    as a design goal (see https://github.com/h2o/picohttpparser).
+
+    From: http://www.boost.org/development/requirements.html
+
+        "Aim first for clarity and correctness; optimization should
+         be only a secondary concern in most Boost libraries."
+
+    As the library matures it will undergo optimization passes, benchmarks
```
Comment (nbougalis) [inline doc/review.qbk]: the comma after "passes" should be a semicolon.

File: doc/review.qbk (hunk 1 above)
Comment (vinniefalco) [inline doc/review.qbk]: willfix

File: doc/review.qbk (hunk 2 above)
Comment (vinniefalco) [inline doc/review.qbk]: willfix

**Comment by vinniefalco** [comment]
Merged to _master_

File: doc/review.qbk (hunk 3 above)
Comment (vinniefalco) [inline doc/review.qbk]: willfix

**Pull request closed without merging**